pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -1541,6 +1541,8 @@ ASTSimpleExpression SimpleExpression() :
                      | LOOKAHEAD({isKeyword("found")}) KEYWORD("FOUND")
                      | LOOKAHEAD({isKeyword("notfound")}) KEYWORD("NOTFOUND")
                      | LOOKAHEAD({isKeyword("rowcount")}) KEYWORD("ROWCOUNT")
+                     | LOOKAHEAD({isKeyword("bulk_rowcount")}) KEYWORD("BULK_ROWCOUNT")
```

Comment:
bulk_rowcount and bulk_exceptions appear to be "implicit cursor attributes": https://docs.oracle.com/en/database/oracle/oracle-database/18/lnpls/implicit-cursor-attribute.html#GUID-5A938EE7-E8D2-468C-B60F-81898F110BE1

However, the syntax is a bit more complicated. I'm not sure, whether adding these keywords here solves the problem already.

Could you provide a sample PLSQL snippet that you want to parse? This can then be used as a testcase to ensure, we can parse it in the future.